src/AppBundle/Controller/CoDeputyController.php

```diff
@@ -83,8 +83,8 @@ public function update(Request $request, $id)
         if (!empty($data['email'])) {
             $originalUser = clone $user;
             $user->setEmail($data['email']);
-            $userService = $this->get('opg_digideps.user_service');
-            $userService->editUser($originalUser, $user);
+            $userService = $this->get('user_service');
+            $userService->editUser($this->getUser(), $originalUser, $user);
```

Comment:
this->getUser can be accessed within  the service cant it? Parameters are a bit confusing? :)

Reply:
this->getUser is the logged in user . In theory it can be injected ( security.token_storage) into the service, but then the service cannot be constructed if the user is not logged. At this stage I moved thing into the service without changing too much